content/en/docs/event-sources/gvisor/_index.md

```diff
@@ -0,0 +1,188 @@
+--- Add the following config, not these instructions.        ---
+
+            "runtimeArgs": [
+                "--pod-init-config=/etc/docker/runsc_falco_config.json"
+
+--- End of added config.                                     ---
+--- Have I told you not to include these instructions.       ---
+
+            ]
+        }
+    }
+}
+```
+
+Then, restart the Docker daemon to let it use the new configuration:
+
+```bash
+$ sudo systemctl restart docker
+```
+
+## Runtime detection in action
+
+Now it's time to put everything together and see how to use Falco to monitor gVisor sandboxes.
+To start monitoring gVisor sandboxes, you can use the `-g` or `--gvisor-config` options, passing the path to the pod init config. Falco uses that config file for two main reasons:
+
+- Extract the path of the UDS that needs to be created
+- Create a trace session for all the already existing gVisor sandboxes. New ones will directly connect to the running Falco instance as we configured in the previous step.
+
+### Run Falco stand-alone
+
+Simply run Falco on the command line:
+
+```bash
+$ sudo falco --gvisor-config=/etc/docker/runsc_falco_config.json
+```
+
+You're now monitoring your gVisor sandboxes.
+
+### Example permament configuration with Systemd
+
+Alternatively, for a more permanent configuration:
+
+```bash
+
+$ sudo mkdir /etc/systemd/system/falco.service.d
+$ cat << EOF | sudo tee /etc/systemd/system/falco.service.d/gvisor.conf
+[Service]
+ExecStartPre=
+ExecStopPost=
+ExecStart=
+ExecStart=/usr/bin/falco --gvisor-config=/etc/docker/runsc_falco_config.json
+PrivateTmp=false
+EOF
+
+$ sudo systemctl daemon-reload
+$ sudo systemctl restart falco
+```
+> The parameter PrivateTmp, set to false, inside the unit config file is needed since /etc/docker/runsc_falco_config.json points to /tmp/gvisor.sock.
+>
+> Changing the /tmp/gvisor.sock file to /run/gvisor.sock would avoid that we have to use that parameter, and the temporary directory would remain private.
+
+Falco will load the configuration indicating it with a line similar to:
+```
+Thu Jul 21 15:41:58 2022: Enabled event collection from gVisor. Configuration path: /etc/docker/runsc_falco_config.json
+```
+
+Now we can run any container with gVisor:
+
+```bash
+$ sudo docker run --runtime=runsc -it ubuntu bash
+```
+
+If all goes well, the container will start properly configured to be monitored by Falco! To test the detection capabilities, try to trigger a simple rule like *Write below binary dir*:
+
+```bash
+$ touch /bin/foo
+```
+
+You will see Falco alerting:
+```
+07:47:42.173335167: Error File below a known binary directory opened for writing (user=root user_loginuid=0 command=touch touch /bin/foo file=/bin/foo parent=bash pcmdline=bash bash gparent=<NA> container_id=f6d77af4ee3d image=ubuntu) container=f6d77af4ee3d pid=8 tid=8
+```
+
+### Falco and gVisor in action
+
+If you don't happen to have the time to try it right now, here is a short video showing every step to follow.
```

Comment:
Could you double check that the instructions in this video are still relevant today? If not we should not include it